spire-server.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: spire-server
+  version: 1.5.3
+  epoch: 0
+  description: The SPIFFE Runtime Environment (SPIRE) server
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - go
+      - build-base
+      - git
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/spiffe/spire/archive/v${{package.version}}/v${{package.version}}.tar.gz
+      expected-sha256: 08a8f97771afae82024b3b407ac4dafe284cd8658ac02855141607b6508b6da6
+  - runs: |
```

Comment:
Maybe this should be a subpackage, so it's clearer they're related, and we only have to bump the version for both in one place.

Reply:
I tried to make it work with sub-packages, but I couldn't. Would you mind helping me to figure this out @imjasonh @tuananh? What am I doing wrong?

Reply:
Created an issue to track adding a validation lint rule https://github.com/wolfi-dev/wolfictl/issues/17

Reply:
that's a perfect issue for @dentrax !